server.go

```diff
@@ -125,17 +96,20 @@ type Server struct {
 	// Res field of the Session you can write an error response
 	// to the client.
 	//
-	// The boolean returned indicates whether the returned subscription
-	// is valid or not. If it is valid, the Provider will receive it
-	// and events will be sent to this client, otherwise the request
-	// will be ended.
+	// The boolean returned indicates whether the given request
+	// should be accepted or not. If it is true, the Provider will receive
+	// a new subscription for the connection and events will be sent
+	// to this client, otherwise the request will be ended.
+	//
+	// Note that you must return any error codes to the client using
+	// the http.ResponseWriter yourself based on the reason for rejection.
```

Comment:
Given that it's incorrect to specify that one _must_ do it, let's rephrase it as follows:
```suggestion
	// Note that OnSession can write the HTTP response code itself, if something other
	// than the implicit 200 OK is desired. This is especially helpful when refusing sessions –
	// if OnSession does not write a response code, clients will receive a confusing 200 OK.
```

Reply:
✅